core/src/test/java/org/elasticsearch/search/aggregations/metrics/cardinality/InternalCardinalityTests.java

```diff
@@ -38,13 +37,21 @@
     private static List<HyperLogLogPlusPlus> algos;
     private static int p;
 
-    @Before
-    public void setup() {
+    @After //we force @After to have it run before ESTestCase#after otherwise it fails
+    @Override
+    public void setUp() {
```

Comment:
Does this need to call super.setUp() as well as it overrides LuceneTestCase#setUp()?